### ISSUE TITLE
The search bar opens too easily

### ISSUE THREAD

**Issue by gabriel-tandil** [SubhamTyagi/Last-Launcher#66]
Nice project. A detail... The search engine opens too easily, it's annoying

**Comment by SubhamTyagi** [comment]
could you elaborate your issue. i couldn't understand what are you referring.

**Comment by unInstance** [comment]
Sometimes you want to get down but you accidentally open the search bar. It happens quite frequently and it is annoying.

Consider having search bar pop up after a horizontal swipe.

The search bar is cool but make the search results stick to the search bar, at the bottom, so that it is suitable for one-handed usage.

**Comment by gabriel-tandil** [comment]
Just like @unInstance say

**Comment by ruevs** [comment]
Exactly! The search should open if one swipes up from no more than the bottom 1/8 of the screen. Now it opens with the bottom 1/4 or so. I open it by mistake so often, that I am considering downgrading to 0.10

**Comment by ruevs** [comment]
I went to read the code and in fact the swipe up gesture does not depend on the location of the swipe, but on the length and speed of the swipe. But I had to read the code to realize this :-)
There is a variable `int swipeMinDistance = 400;` which I think is in pixels. That would make a fast enough movement of more than 400/1920 = 1/5 of the screen a swipe (What about 1280x720 or high resolution screens? I do not know anything about Android development) .

When swiping with my thumb holding the phone with one hand (from the bottom of the screen) it seems I routinely do long and fast enough swipes to open the search.

I would make a swipe at least 1/3 screen height.
